kpi/tests/api/test_api_environment.py

```diff
@@ -116,6 +117,7 @@ def setUp(self):
             'custom_password_localized_help_text': markdown(
                 I18nUtils.get_custom_password_help_text()
             ),
+            'terms_of_service__sitewidemessage__exists' : SitewideMessage.objects.filter(slug='terms_of_service').exists(),
```

Comment:
This appears to match the implementation logic — and so it doesn't exactly test the logic itself (or catch a regression)

Could we set up a couple of simple/exact test cases to cover each possibility? 

- (1) set a sitewide message called terms_of_service to a static value, assert env's terms_of_service__sitewidemessage__exists reads true
- (2) remove or don't set a sitewide message called terms_of_service, assert env's terms_of_service__sitewidemessage__exists reads false